pkg/apis/networkservicemesh.io/v1/types.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/ligato/networkservicemesh/netmesh/model/netmesh"
+)
+
+// Constants to register CRDs for our resources
+const (
+	NSMGroup           string = "networkservicemesh.io"
```

Comment:
This has to have a "." in the name or trying to create the CRD results in an error such as this:

```
    time="2018-05-30 13:58:58.32253" level=error msg="Error loading core: plugin CRD: AfterInit error 'CustomResourceDefinition.apiextensions.k8s.io \"networkserviceendpoints.networkservicemesh\" is invalid: spec.group: Invalid value: \"networkservicemesh\": should be a domain with at least one dot', took 37.918062ms" loc="core/event_loop.go(29)" logger=agentcore
```